lib/msf/core/auxiliary/mqtt.rb

```diff
@@ -24,11 +24,11 @@ def initialize(info = {})
       end
 
       def setup
```

Comment:
Is this dead code? Looks like the only use of this mixin is

```
modules/auxiliary/scanner/mqtt/connect.rb
```

Which doesn't call this code as it uses the `include Msf::Auxiliary::AuthBrute` which doesn't call `super` in its `setup` method? 👀 

```
>> self.class
=> Msf::Modules::Auxiliary__Scanner__Mqtt__Connect::MetasploitModule
>> self.method(:setup).source_location
=> 
["/Users/adfoster/Documents/code/metasploit-framework/lib/msf/core/auxiliary/auth_brute.rb",
 48]
```

I'm guessing it was previously tested, but broken accidentally? 🤷 
https://github.com/rapid7/metasploit-framework/commit/ed5f177fcd70248c607a4df17f526dec39fe5826#diff-b9616f8dab41c6c04e2c53235d97941154ff73e87cbad03308b5cffeeff5063bL10-R14

Reply:
Removing that AuthBrute mixin works:
```
>> self.method(:setup).source_location
=> 
["/Users/adfoster/Documents/code/metasploit-framework/lib/msf/core/auxiliary/mqtt.rb",
 26]
```

And shows the constant exception now:
```
msf6 auxiliary(scanner/mqtt/connect) > run
[-] Auxiliary failed: Msf::OptionValidateError One or more options failed to validate: RHOSTS.
msf6 auxiliary(scanner/mqtt/connect) > run rhosts=127.0.0.1

[-] 127.0.0.1:1883        - Auxiliary failed: NameError uninitialized constant Msf::Auxiliary::MQTT::Failure
[-] 127.0.0.1:1883        - Call stack:
[-] 127.0.0.1:1883        -   /Users/user/Documents/code/metasploit-framework/lib/msf/core/auxiliary/mqtt.rb:27:in `setup'
[*] Auxiliary module execution completed
```